packages/server/nodemon.json

```diff
@@ -1,13 +1,19 @@
 {
   "watch": [
     "src",
-    "../backend-core",
-    "../pro",
-    "../types",
-    "../shared-core",
-    "../string-templates"
+    "../backend-core/src",
+    "../pro/src",
+    "../types/src",
+    "../shared-core/src",
+    "../string-templates/src"
```

Comment:
Specifying only the `src` directories appears to fix the intermittent issues of endless restarts via nodemon (due to a loop of nodemon > build > something changes > nodemon restart > build... etc).

Reply:
Good catch!

Reply:
Amazing!